api/v1alpha1/datadogagent_types.go

```diff
@@ -358,6 +374,33 @@ type ProcessSpec struct {
 	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
 }
 
+// OrchestratorExplorerConfig contains the orchestrator explorer configuration.
+// The orchestratorExplorer runs in the process-agent and DCA.
+// +k8s:openapi-gen=true
+type OrchestratorExplorerConfig struct {
+	// Enable this to activate live kubernetes monitoring.
+	// ref: https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources
+	//
+	// +optional
+	Enabled *bool `json:"enabled,omitempty"`
+
+	// +optional
+	// Deactivate this to stop the scrubbing of sensitive container data (passwords, tokens etc. ).
+	ContainerScrubbingEnabled *bool `json:"containerScrubbingEnabled,omitempty"`
```

Comment:
Do you think in the future you will introduce `Scrubbling` option? 
if yes maybe you can create a `ContainerScrubbling` struct

```golang
type ContainerScrubbling struct {
    Enabled *bool `json:"enabled,omitempty"`
}
```

Reply:
We were thinking about the possibility of adding multiple features to Container something like this

```go
type OrchestratorContainerOptions struct {
    ScrubbingEnabled *bool `json:"enabled,omitempty"`
    Feature2
    Feature3
}
```

But this is something we may add in the further future. Do you prefer to handle that case in this PR?

Reply:
I agree with other features enabling.

But for this specific feature, do you think we can have other sub-settings, or maybe `scrubbing` other resources like `configmap` `secret`?

in this case maybe a struct like
```yaml
    scrubbing:
       container: true
       configMap: true
```

could make more sens. WDYT?

Reply:
ah this makes sense. I don't think we had that granularity in mind while designing this feature. It certainly makes sense to invest more time into that in the future. I think we could leave it currently as it is and revisit if needed? The refactoring should not be too much work. 🤔 
But I don't have a big opinion either. 

Maybe @hkaj has an opinion on it?